back/db/views/analytics_fact_participations_v01.sql

```diff
@@ -0,0 +1,63 @@
+-- Analytics view for all types of participation
```

Comment:
Not a big deal, but I find the name of the view a bit misleading because when we talk about 'types of participations', it usually includes posting ideas, volunteering, taking a survey, and creating a basket, whereas this view is focused on vote counts.

Reply:
yeah, we are not using that fact table in this first iteration but we will be using it soon and it will grow to include all posible participation types. for now its just voting.